stable/pgadmin/templates/pvc.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
   {{ toYaml . | indent 4 }}
   {{- end }}
 spec:
+{{- if .Values.persistence.storageClassName }}
+  storageClassName: {{ .Values.persistence.storageClassName }}
+{{- end }}
```

Comment:
Perhaps this would be better:
```
{{- if .Values.persistentVolume.storageClass }}
{{- if (eq "-" .Values.persistentVolume.storageClass) }}
  storageClassName: ""
{{- else }}
  storageClassName: "{{ .Values.persistentVolume.storageClass }}"
{{- end }}
```

Reply:
I'd find it more clear to no work with setting a value to "-": A defined values goes into the deployment, an undefined does not.